### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Access Keys_WAT: Color Contrast ratio for "White" foreground and "Blue" background is "3.1.1 which is less than 4.5:1.

### ISSUE THREAD

**Issue by Accessibility56** [Microsoft/IIS.WebManager#64]
GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #.NETWebsites; #ColorContrast; #WCAG1.4.3; #C+E-H; #Edge

**Environment Details:**
**URL:** https://localhost:55539/security/tokens
**Browser Details:** Edge(42.17134.1.0)
Web Accessibility Toolbar, Color Contrast

**Repro Steps:**
1.Open "Microsoft IIS" application using the URL: https://manage.iis.net/
2.Download and install "Microsoft IIS Administration" application.
3.Tab till "Skip This" button and Press Enter.
4.Tab till "Get Access Token" link and Press Enter.
5.Enable Color Contrast Analyzer.
7.Verify Color contrast ratio for white foreground and blue background.
  
**Actual Result:**
Color Contrast ratio for "White" foreground and "Blue" background is "3.1.1 which is less than 4.5:1.

**Expected Result:**
Foreground to background color contrast ratio should be >=4.5:1.

**User Impact:**
Low vision users find difficulty in viewing the controls if color contrast ratio is not proper.

**Attachment:**
![bug_64_access keys_color contrast issue](https://user-images.githubusercontent.com/38523250/44568803-ecd2be00-a795-11e8-9da4-895db1505d8a.png)

**Comment by jhkimnew** [comment]
Fixed

**Comment by fsteffi** [comment]
#A11ysev2;

**Comment by fsteffi** [comment]
#closed;
manage.iis.net is deprecated. The "Manage IIS" button on the [www.iis.net](http://www.iis.net/) home page will  be removed.
